jointeam.html

```diff
@@ -326,8 +326,7 @@ <h2 class = "subtitle-heading">Current Volunteer Postings</h2>
         <div class="column">
           <div class="container-vertical">
             <div class="slide">
-              <div id="administrative-assistant-option" class="tab-vertical" role="button" tabindex = "0" aria-haspopup="true">Administrative Assistant</div>
-              <div id="board-member-option" class="tab-vertical" role="button" tabindex = "0" aria-haspopup="true">Board Member <span  id ="two"class="size sizeone">+</span></div>
+             
```

Comment:
You have to delete the entire `<div class="slide">...</div>` from line 328 to 334. Otherwise, there is a blank space above "Administrative Assistant".